tests/integration/targets/lookup_aws_secret/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
+shippable/aws/group2
```

Comment:
```suggestion
```
We no longer use shippable (I only just removed these lines from existing test suites in this past week :)